_dashboards/drag-drop-wizard.md

```diff
@@ -1,56 +1,71 @@
 ---
 layout: default
-title: Drag and drop wizard
+title: Drag-and-drop wizard
 nav_order: 8
 ---
 
-The drag and drop visualization wizard an experimental feature with OpenSearch 2.3. Therefore, we do not recommend the use of drag and drop wizard in a production environment. For updates on the progress of drag and drop or if you want leave feedback that could help improve the feature, see the [Drag and drop git issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1157). 
+The drag-and-drop visualization wizard an experimental feature with OpenSearch 2.3. Therefore, we do not recommend the use of drag-and-drop wizard in a production environment. For updates on the progress of drag and drop or if you want leave feedback that could help improve the feature, see the [Drag and drop git issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1157). 
 {: .warning}
 
-# Drag and drop wizard
+# Drag-and-drop wizard
 
-You can use the drag and drop visualization wizard in OpenSearch Dashboards to create your data visualizations easily with just a few clicks or a single mouse gesture. 
+You can use the drag-and-drop visualization wizard in OpenSearch Dashboards to create your data visualizations easily with just a few clicks or a single mouse gesture. 
 
-The drag and drop visualization wizard:
+The drag-and-drop visualization wizard:
 
-* Gives you an immediate view of your data without the need to preselect the visualization output 
-* Gives you the flexibility to change visualization types and index patterns on the fly
-* Gives you the ability to easily navigate between multiple screens 
+* Gives you an immediate view of your data without the need to preselect the visualization output. 
+* Gives you the flexibility to change visualization types and index patterns quickly.
+* Gives you the ability to easily navigate between multiple screens. 
 
 <img src="{{site.url}}{{site.baseurl}}/images/drag-drop-ui.png" alt="Drag and drop user interface">
 
+## Try out the wizard
 
-## Enable the wizard
+You can try out the wizard locally or in the [OpenSearch playground](https://playground.opensearch.org/app/home#/).
```

Comment:
just a URL typo here: 
need to point directly to the wizard in the playground: 
https://playground.opensearch.org/app/wizard